include/gpio.h

```diff
@@ -202,6 +202,12 @@ static inline int gpio_pin_configure(struct device *port, u32_t pin,
 
 /**
  * @brief Write the data value to a single pin.
+ *
+ * Write logical data value to a pin, taking into account GPIO_ACTIVE_LOW flag.
+ * If an output was configured as Active Low, writing value 1 to a pin will set
+ * it to low output state. Otherwise writing value 1 to a pin will set it to
+ * high output state.
```

Comment:
This isn't what those flags are intended to convey, nor does this describe how this function actually behaves (unless I am *very* confused...).

Reply:
I am against changing the definition of how `gpio_pin_write` and `gpio_pin_read` are expected to behave. This would require alteration of all existing implementations of GPIO drivers (as these flags cannot be just not supported).
In my opinion the `GPIO_ACTIVE_*` flags should be handled outside of particular GPIO drivers, for example in wrappers as I proposed in https://github.com/zephyrproject-rtos/zephyr/pull/11880#issuecomment-461011334.